code/drasil-gool/GOOL/Drasil/LanguageRenderer/CSharpRenderer.hs

```diff
@@ -245,20 +230,20 @@ instance VariableSym CSharpCode where
 
 instance ValueSym CSharpCode where
   type Value CSharpCode = ValData
-  litTrue = liftA2 mkVal bool (return litTrueD)
-  litFalse = liftA2 mkVal bool (return litFalseD)
-  litChar c = liftA2 mkVal char (return $ litCharD c)
-  litFloat v = liftA2 mkVal float (return $ litFloatD v)
-  litInt v = liftA2 mkVal int (return $ litIntD v)
-  litString s = liftA2 mkVal string (return $ litStringD s)
+  litTrue = litTrueD
+  litFalse = litFalseD
```

Comment:
one of the things to look at, in the future, will be if all `instance`s of (say) `litFalse` all dispatch to the generic one -- and then maybe remove it from the `ValueSym` class altogether, since it is never overloaded.

in some case, we'll want to keep them anyways. But then a 'better' mechanism might be to use default in the class declaration itself, so that the actual instances can be made much shorter.